test/test_account_operations.sh

```diff
@@ -17,4 +17,4 @@ if [[ ! "$RESULT" =~ $EXPECTED ]]; then
     exit 1
 fi
 
-../bin/near delete_account $testaccount test.near
+../bin/near delete $testaccount test.near
```

Comment:
Is this related to the `yargs` changes? Should it be submitted as a separate PR?

Reply:
never updated from:
https://github.com/nearprotocol/near-shell/commit/434532f9e28ce38ebe8b209b271e1d107b4fdaa3#diff-22d5d2b57c69d2e1ac054c4d0f744ff1L53-R111

Reply:
Ok, so this is a broken thing that just needed to be fixed, and the fix is small enough that it may as well get fixed as part of this larger PR?